### ISSUE TITLE
docker 1.6

### ISSUE THREAD

**Issue by jumanjiman** [coreos/bugs#331]
i'd love to see docker 1.6.
i looked for roadmap or open issues for $subject but did not see anything relevant.

is there an eta to add $subject?

**Comment by jumanjiman** [comment]
example use case: dev environment uses remote docker api against coreos remote host, but
dev env and remote host use different API versions.

```console
user@devenv:~$ grep -i docker_host .bashrc 
export DOCKER_HOST='tcp://192.168.254.162:2375'

user@devenv:~$ docker run --rm -it alpine:latest sh
Unable to find image 'alpine:latest' locally
FATA[0000] Error response from daemon: client and server don't have same version (client : 1.18, server: 1.17) 
```

**Comment by ayumi** [comment]
+1 for Docker 1.6 please.

Would love to start using Registry 2.0.

**Comment by mischief** [comment]
docker 1.6 is merged with coreos/coreos-overlay#1220. it will be available beginning with the next alpha image on april 30.

**Comment by ayumi** [comment]
awesome, thanks! :)
